modules/creating-the-vsphere-windows-vm-golden-image.adoc

```diff
@@ -3,87 +3,91 @@
-
-. Create the `C:\Users\Administrator\.ssh\authorized_keys` file in the Windows VM containing the public key that corresponds to the private key that resides in the secret you created in the `openshift-windows-machine-config-operator` namespace. The private key of the secret was created when first installing the Windows Machine Config Operator (WMCO) to give {product-title} access to Windows VMs. The `authorized_keys` file is used to configure SSH in the Windows VM.
-
-. Configure SSH on the Windows VM by running the following Powershell script:
+. Create a new virtual machine in the vSphere client from the Windows Server 2004 ISO image. See link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.hostclient.doc/GUID-7834894B-DD17-4D59-A9BF-A33D02478521.html[VMware's documentation] for more information.
 +
-[source,posh]
-----
-# Powershell script to configure SSH on vSphere Windows VMs
-Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
-$firewallRuleName = "ContainerLogsPort"
-$containerLogsPort = "10250"
-New-NetFirewallRule -DisplayName $firewallRuleName -Direction Inbound -Action Allow -Protocol TCP -LocalPort $containerLogsPort -EdgeTraversalPolicy Allow
-Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
-Install-Module -Force OpenSSHUtils
-Set-Service -Name ssh-agent -StartupType 'Automatic'
-Set-Service -Name sshd -StartupType 'Automatic'
-Start-Service ssh-agent
-Start-Service sshd
-$pubKeyConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PubkeyAuthentication yes','PubkeyAuthentication yes'
-$pubKeyConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$passwordConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace '#PasswordAuthentication yes','PasswordAuthentication yes'
-$passwordConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$authFileConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace 'AuthorizedKeysFile __PROGRAMDATA__/ssh/administrators_authorized_keys','#AuthorizedKeysFile __PROGRAMDATA__/ssh/administrators_authorized_keys'
-$authFileConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-$pubKeyLocationConf = (Get-Content -path C:\ProgramData\ssh\sshd_config) -replace 'Match Group administrators','#Match Group administrators'
-$pubKeyLocationConf | Set-Content -Path C:\ProgramData\ssh\sshd_config
-Restart-Service sshd
-New-item -Path $env:USERPROFILE -Name .ssh -ItemType Directory -force
-----
+[IMPORTANT]
+====
+The Windows Server 2004 VM must include the link:https://support.microsoft.com/en-us/help/4565351/windows-10-update-kb4565351[Microsoft patch KB4565351]. This patch is required to set the VXLAN UDP port, which is required for clusters installed on vSphere. This patch is not available for the `Windows Server 1809` VM image.
```

Comment:
See [Select a compatible Windows Server version](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/vsphere-golden-image.md#1-select-a-compatible-windows-server-version)

Reply:
On a tangential note, I discovered during a `sig-windows` call that `Windows Server 1809` also refers to a SAC release. It just so happened that the 2019 LTSC release matched 1809. But saying 1809 apparently implies the SAC version as it is possible to upgrade from 1809 to 2004 etc but not from LTSC 2019 to 2004. You can only upgrade LTSC to another LTSC version like 2022.